planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/scene_crosswalk.cpp

```diff
@@ -382,7 +382,7 @@ std::optional<StopFactor> CrosswalkModule::checkStopForCrosswalkUsers(
   // Check if ego moves forward enough to ignore yield.
   const auto & p = planner_param_;
   if (!path_intersects.empty()) {
-    const double base_link2front = planner_data_->vehicle_info_.max_longitudinal_offset_m;
```

Comment:
Delete this line is sufficient because the outer value is same as this value

Reply:
I have removed this line as you indicated.